azure-functions/acbs-function/helpers/mandatoryFields.js

```diff
@@ -1,5 +1,12 @@
-const findMissingMandatory = (data, mandatoryFields) =>
-  mandatoryFields.map((mf) => (data[mf] !== null || data[mf] !== undefined || data[mf] !== '' ? '' : mf)).filter((mf) => mf !== '');
+/**
+ * Finds missing mandatory fields in the provided data object.
+ *
+ * @param {Object} data - The data object to check for missing fields.
+ * @param {Array<string>} mandatoryFields - An array of field names that are mandatory.
+ * @returns {Array<string>} - An array of missing mandatory field names. If all mandatory fields are present, returns an empty array.
+ */
+
+const findMissingMandatory = (data, mandatoryFields) => mandatoryFields.map((field) => (data[field] ? false : field)).filter((field) => field);
```

Comment:
This fails on falsy values -- ie
```
aTemplate = ['totalToPayThisMonth', 'shouldDoSomethingImportant']
```

if we gave it the following:
```
{
    totalToPayThisMonth: 0,
    shouldDoSomethingImportant: false
}
```

it will return that the fields are missing.


Working example:
```
const findMissingMandatory = (data, mandatoryFields) => mandatoryFields.map((field) => (data[field] ? false : field)).filter((field) => field);

const data = {
    tester: 0,
    tester2: false
}

console.log(findMissingMandatory(data, ['tester', 'tester2', 'tester3']));
```